opengrok-web/src/main/java/org/opengrok/web/WebappListener.java

```diff
@@ -53,13 +57,19 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(WebappListener.class);
 
+    private static final String TIMER_ATTR_NAME = "request_start";
+    private static Timer requestTimer;
```

Comment:
Why `static`?

Reply:
hmm, not necessary, will remove. I don't actually know how the `listener-class` is instantiated.